src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -81,6 +81,24 @@ tenant_tunnel_network_vlanid=130
 tenant_vlan_range=201:250
 
 
+[MTU Settings]
+# if the "global" option is selected, mtu value for all networks will be set  
+# to the value provided in mtu_size_global_default i.e. 1500-9000
+# if the "per_network" option is selected, the user should provide mtu value for each network in the range of 1500 and 9000.
+# for public and floating api network, mtu greater than 1500 is only supported if jumbo frames are enabled 
```

Comment:
Capitalize for.  I would say "For public api and floating ip networks,".  Floating isn't the api network per se